packages/spark-core/settings/_settings.scss

```diff
@@ -522,7 +493,7 @@ $dropdown-shadow: 0 5px 10px rgba(0, 0, 0, 0.05) !default;
 
 //
 // Modals
-///
```

Comment:
@yodasw16 changed these all to /// from //. Did you mean to change these back or was it an editor thing?